e2e/interchaintestv8/ethereum/utils_test.go

```diff
@@ -0,0 +1,136 @@
+package ethereum_test
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+
+	"github.com/srdtrk/solidity-ibc-eureka/e2e/v8/ethereum"
+)
+
+func TestGetEthAddressFromStdout(t *testing.T) {
```

Comment:
Lol, this function seems a bit extreme but alright. We might have the percentage of golang exceed that of Solidity in this repo after this PR :smiling_face_with_tear: